roles/reset/tasks/main.yml

```diff
@@ -310,6 +329,21 @@
   tags:
     - files
 
+- name: reset | remove containerd files
+  file:
+    path: "{{ containerd_bin_dir }}/{{ item }}"
+    state: absent
+  with_items:
+    - containerd
+    - containerd-shim
+    - containerd-shim-runc-v1
+    - containerd-shim-runc-v2
+    - ctr
```

Comment:
Missed `{{ containerd_bin_dir }}/runc`

Reply:
This may actually need to be changed to make molecule tests happy about the idempotency of the download/unarchive tasks.

Reply:
Latest changes properly puts `runc` under it's own location.